modules/create_folders.nf

```diff
@@ -1,18 +1,40 @@
-// builds or updates the subfolders with log and predictions files
+// This Nextflow script contains 2 tasks:
+// 1. CREATE_FOLDERS : Takes in (submission_id, create_or_update) and project_name to create
+// root and subfolders for the designated Challenge Project on Synapse
+// 2. UPDATE_FOLDERS : Takes in (submission_id, create_or_update), project_name, and predictions_file_path
+// fed by RUN_DOCKER, to update the subfolders created in CREATE_FOLDERS
+
 process CREATE_FOLDERS {
     secret "SYNAPSE_AUTH_TOKEN"
     container "sagebionetworks/synapsepythonclient:v4.0.0"
 
     input:
-    tuple val(submission_id), val(build_or_update)
+    tuple val(submission_id), val(create_or_update)
     val project_name
-    val ready
 
     output:
     val "ready"
 
     script:
     """
-    create_folders.py '${project_name}' '${submission_id}' '${build_or_update}'
+    create_folders.py '${project_name}' '${submission_id}' '${create_or_update}'
     """
 }
+
+process UPDATE_FOLDERS {
```

Comment:
CREATE_FOLDERS and UPDATE_FOLDERS required different inputs, so I went the route of creating separate tasks for each of them. I still kept them stored under the same nf script `create_folders.nf` since both of them are calling the same Python script in the script section, and UPDATE_FOLDERS is technically creating new folders (re-creating them), so it felt approrpiate to leave it under a script called `create_folders`